lib/adal-angular.js

```diff
@@ -179,7 +179,7 @@
 
                 };
 
-                var loginHandler = function () {
+                var loginHandler = function (returnUrl) {
                     _adal.info('Login event for:' + $location.$$url);
```

Comment:
if loginStartPage is passed in the function, the _adal.info should print the loginStartPage instead of the absolute url used earlier. Please change this to:
var loginUrl = loginStartPage || $location.$$url;
                    if (_adal.config && _adal.config.localLoginUrl) {
                        _adal.info('Login event for:' + _adal.config.localLoginUrl);
                        $location.path(_adal.config.localLoginUrl);
                    }
                    else {
                        // directly start login flow
                        _adal.info('Start login at:' + loginUrl);
                        $rootScope.$broadcast('adal:loginRedirect');
                        _adal.login(loginUrl);
                    }